cli/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
-## 13.8.1
+## 13.9.0
```

Comment:
There should be a new section for `13.9.0` since `13.8.1` has already been released.

Reply:
addressed in https://github.com/cypress-io/cypress/pull/29312/commits/122cd50cf0a3aae0418874dba7b6146097866532